### PR TITLE
[flink] Fetching schema in Kafka table synchronization should first seek to beginningOffsets

### PR DESCRIPTION
**Pull request by ChenShuai1981**
…chema otherwise perhaps can NOT fetch records in little traffic case

[flink]
### Purpose

Linked issue: when run kafka sync table action it throws can NOT get metadata of topic error

Seek to beginningOffsets and enlarge poll duration when fetch kafka schema otherwise perhaps can NOT fetch records in little traffic case

### API and Format

NO

### CODE REVIEW

**Comment by ChenShuai1981** [comment]
Close and Refire another PR

File: paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/kafka/KafkaSchema.java [hunk 1]
```diff
@@ -99,7 +100,15 @@ private static KafkaConsumer<String, String> getKafkaEarliestConsumer(
...
-        consumer.assign(Collections.singletonList(new TopicPartition(topic, firstPartition)));
+        Collection<TopicPartition> topicPartitions =
+                Collections.singletonList(new TopicPartition(topic, firstPartition));
+        consumer.assign(topicPartitions);
+
+        Map<TopicPartition, Long> beginningOffsets = consumer.beginningOffsets(topicPartitions);
+        for (TopicPartition tp : topicPartitions) {
+            Long offset = beginningOffsets.get(tp);
+            consumer.seek(tp, offset);
+        }
```
Comment (yuzelin) [inline paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/kafka/KafkaSchema.java:111]: How about just using `consumer.seekToBeginning(topicPartitions);` ?

Comment (ChenShuai1981) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/kafka/KafkaSchema.java]: > How about just using `consumer.seekToBeginning(topicPartitions);` ?

Yes, that's more straight forward, adopt your suggestion.

**Review by yuzelin — APPROVED** [review]
+1

**Pull request merged**